src/main/java/de/cismet/cids/custom/wunda_blau/search/server/QsgebStatusLightweightSearch.java

```diff
@@ -0,0 +1,241 @@
+     */
+    public QsgebStatusLightweightSearch() {
+        this.searchInfo = new SearchInfo(
+                this.getClass().getName(),
+                this.getClass().getSimpleName(),
+                "Builtin Legacy Search to delegate the operation getLightweightMetaObjectsByQuery to the cids Pure REST Search API.",
+                Arrays.asList(
+                    new SearchParameterInfo[] {
+                        new MySearchParameterInfo(FIELD__STATUS_ID, Type.INTEGER),
+                        new MySearchParameterInfo("searchFor", Type.UNDEFINED),
+                        new MySearchParameterInfo("representationPattern", Type.STRING, true),
+                        new MySearchParameterInfo("representationFields", Type.STRING, true)
+                    }),
+                new MySearchParameterInfo("return", Type.ENTITY_REFERENCE, true));
+    }
+
+    /**
+     * Creates a new StatusArtLightweightSearch object.
+     *
+     * @param  searchFor              DOCUMENT ME!
+     * @param  representationPattern  DOCUMENT ME!
+     * @param  representationFields   DOCUMENT ME!
+     */
+    public QsgebStatusLightweightSearch(final SearchFor searchFor,
+            final String representationPattern,
+            final String[] representationFields) {
+        this();
+        setSearchFor(searchFor);
+        setRepresentationPattern(representationPattern);
+        setRepresentationFields(representationFields);
+    }
+
+    //~ Methods ----------------------------------------------------------------
+
+    @Override
+    public void initWithConnectionContext(final ConnectionContext connectionContext) {
+        this.connectionContext = connectionContext;
+    }
+
+    @Override
+    public ConnectionContext getConnectionContext() {
+        return connectionContext;
+    }
+
+    @Override
+    public Collection performServerSearch() throws SearchException {
+        final SearchFor searchFor = getSearchFor();
+        final Integer statusId = getStatusId();
+
+        if (searchFor == null) {
+            throw new SearchException("searchFor has to be set");
+        }
+
+        final MetaService metaService = (MetaService)this.getActiveLocalServers().get("WUNDA_BLAU");
+        if (metaService == null) {
+            final String message = "Lightweight Meta Objects By Query Search "
+                        + "could not connect ot MetaService @domain 'WUNDA_BLAU'";
+            LOG.error(message);
+            throw new SearchException(message);
+        }
+
+        final Collection<String> conditions = new ArrayList<>();
+        if (statusId != null) {
+            //conditions.add(String.format("status = %d", statusId));
+            switch (statusId){
+                case 0:
+                    conditions.add(String.format("id = %d", 0));
+                    conditions.add(String.format("id = %d", 1));
+                    conditions.add(String.format("id = %d", 4));
```

Comment:
No String.format necessery =>
`conditions.add("id = 0");`
...is less complicated and more readable.